e2e/testsuite/testsuite.go

```diff
@@ -361,24 +392,6 @@ func (s *E2ETestSuite) newInterchain(ctx context.Context, relayers []ibc.Relayer
 		}
 	}
 
-	s.startRelayerFn = func(relayer ibc.Relayer) {
-		// depending on the test, the path names will be different.
-		// whenever a relayer is started, it should use the paths associated with the test the relayer is running in.
-		pathNames, ok := s.testPaths[s.T().Name()]
-		s.Require().True(ok, "path names not found for test %s", s.T().Name())
-
-		err := relayer.StartRelayer(ctx, s.GetRelayerExecReporter(), pathNames...)
-		s.Require().NoError(err, fmt.Sprintf("failed to start relayer: %s", err))
-
-		var chainHeighters []test.ChainHeighter
-		for _, c := range chains {
-			chainHeighters = append(chainHeighters, c)
-		}
-
-		// wait for every chain to produce some blocks before using the relayer.
-		s.Require().NoError(test.WaitForBlocks(ctx, 10, chainHeighters...), "failed to wait for blocks")
-	}
-
```

Comment:
the restructuring of code makes it so that this custom fn is no longer required.